config/database.js

```diff
@@ -1,12 +1,22 @@
 const mongoose = require('mongoose')
 
-mongoose.connect('mongodb://127.0.0.1/WeAreYourTeam')
+const dbURI = 'mongodb://127.0.0.1/WeAreYourTeam' //should put it to .env
+
+mongoose.connect(dbURI)
 
 mongoose.Promise = Promise
 
 const db = mongoose.connection
 
-// Bind connection to error event (to get notification of connection errors)
+db.on('connected', () => {
+  console.log('MongoDB connection connected to ' + dbURI)
```

Comment:
Our linter is not so happy with us using `console.log`. For now disable the `no-console` rule for this file, just so it shuts up.

Maybe later we can consider using a proper logging library

Reply:
Yeah, honestly we can disable the rule all together on the back-end. I just copied what we use at my current workplace (which is one big repository for both front and back-end) but in this case, logging isn't a big deal.

Reply:
Added disable no-console in the top of file